customisation/customisation.md

```diff
@@ -4,3 +4,17 @@ has_children: true
 title: Customisation
 description: Machine translation customisation
 ---
+
+{% assign customizable_apis = site.data.apis | where: "customization_languages", true | where: "glossary", true | concat: site.data.apis | where: "formality", true | concat: site.data.apis | where: "adaptive", true %}
+
+{{ customizable_apis | size }} APIs support customisation.
```

Comment:
We should probably use international English spellings in code variable names too, just to be consistent.

So `customisable_apis`.